frontend/src/components/layout/Header/Header.tsx

```diff
@@ -0,0 +1,50 @@
+// import React, { useEffect, useState } from 'react';
+
+import flower from '../../../assets/icons/ML_flower-icon.png';
+import login from '../../../assets/icons/ML_login_icon.png';
+import logout from '../../../assets/icons/ML_logout-icon.png';
+import moon from '../../../assets/icons/ML_moon-icon.png';
+import sun from '../../../assets/icons/ML_sun-icon.png';
+import my from '../../../assets/icons/ML_my-icon.png';
+import pencil from '../../../assets/icons/ML_pencil-icon.png';
+
+import { HeaderIconText } from "../../common/HeaderIconText";
+
+const Header = ({ blogName }: { blogName: string }) => {
+    // 추후 로그인 여부 확인
+    // const [isLogin, setIsLogin] = useState(false);
+
+    const checkIsLogin = () => {
+        return true;
+    }
+
+    return (
+        <header className="w-full h-16 flex bg-transparent font-PyeongChangPeace ">
+            <div className="basis-[40%]" />
```

Comment:
P4: `header`를 우측 정렬하면 L#23줄이 없어도 될 것 같습니다.
```suggestion
        <header className="w-full h-16 flex justify-end bg-transparent font-PyeongChangPeace ">
```

Reply:
오호 해당 방식으로도 동작하는군요! 수정하겠습니다.